lib/Rex/Commands/Iptables.pm

```diff
@@ -55,7 +55,25 @@ Only I<open_port> and I<close_port> are idempotent.
          accept          => "POSTROUTING",
          "out-interface" => "eth0",
          jump            => "MASQUERADE";
- 
+
+   # Version of IP can be specified in the first argument
+   # of any function: -4 or -6 (if no version specified,
+   # -4 implies)
```

Comment:
Kudos for the documentation bits! I think `defaults to -4` might be slightly better wording here.